modules/oe_starter_content_event/config/install/core.entity_view_display.node.oe_sc_event.default.yml

```diff
@@ -41,13 +41,12 @@ content:
     weight: 6
     region: content
   oe_featured_media:
-    type: entity_reference_entity_view
+    type: oe_featured_media_label
     label: above
     settings:
-      view_mode: default
       link: true
     third_party_settings: {  }
-    weight: 3
+    weight: 102
```

Comment:
The weight seems off, the field was supposed to show higher in the page. Better re-save the form display and re-export.